RandBLAS/trig_skops.hh

```diff
@@ -0,0 +1,493 @@
+#include "RandBLAS/base.hh"
+#include "RandBLAS/exceptions.hh"
+#include "RandBLAS/random_gen.hh"
+#include <RandBLAS/sparse_skops.hh>
+
+#include <Random123/philox.h>
+#include <blas.hh>
+#include <lapack.hh>
+#include <omp.h>
```

Comment:
RandBLAS doesn't have OpenMP as a hard dependency. So you need any ``include <omp.h>`` and any OpenMP function calls to be wrapped with compiler preprocessor directives.

Reply:
I didn't end up using OpenMP anywhere else (so-far), so I've dropped the dependency for now!